### PR TITLE
README: Add 'Getting started' directions

### PR DESCRIPTION
**Pull request by cassidyjames**
Based on #29, but a bit expanded and hopefully addressing concerns on that PR. Omits docs instructions for now as I'm not familiar with how that works.

### CODE REVIEW

**Comment by cassidyjames** [comment]
I would like to add a section like this at the end, but am unable to actually build the docs locally for some reason, so I'm omitting for now:

>### Docs
>
>libportal can optionally generates docs with the `gtk_doc` option which defaults to `true`. View the resulting documentation in the `doc/html/` subdirectory of your build directory, e.g.:
>
>```
>gio open _build/doc/html/index.html
>```

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]
Mostly looks good, clearly an improvement over the single sentence we currently have, but could you please limit the line width to 80 characters?

File: README.md [hunk 1]
```diff
@@ -2,3 +2,75 @@ libportal - Flatpak portal library
...
+```
+ninja -C _build
+```
+
+## Optional components
+
+### `portal-test` and `portal-test-qt`
+
+Set the `build-portal-test` or `build-portal-test-qt` option to `true` and build
+the project to compile the corresponding binary. Then run the binary, e.g.:
```
Comment (GeorgesStavracas) [inline README.md:63]: This was recently changed to `portal-tests`, and it is passed as `-Dportal-tests=gtk3,gtk4,qt` (or any combination, or none to disable all tests)

File: README.md [hunk 2]
```diff
@@ -2,3 +2,75 @@ libportal - Flatpak portal library
...
+
+```
+./_build/portal-test/portal-test
```
Comment (GeorgesStavracas) [inline README.md:66]: And correspondingly to the previous comment, the file paths changed to e.g. `portal-test/gtk3/portal-test-gtk3`

File: README.md [hunk 3]
```diff
@@ -2,3 +2,75 @@ libportal - Flatpak portal library
...
+with:
+
+```
+meson configure _build
+```
+
+To change an option, re-configure the project:
+
+```
+meson configure _build -Dbuild-portal-test=true
```
Comment (GeorgesStavracas) [inline README.md:43]: See previous comments; it's now `-Dportal-tests=...`

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]
Okay, sorry I didn't catch this in the previous round. I think it's good to land after these adjustments.

**Review by GeorgesStavracas — APPROVED** [review]
Thank you so much!

**Pull request merged**